test/unit/org/apache/cassandra/index/sai/SAITester.java

```diff
@@ -330,6 +330,8 @@ protected boolean verifyChecksum(IndexContext context)
         for (SSTableReader sstable : cfs.getLiveSSTables())
         {
             IndexDescriptor indexDescriptor = IndexDescriptor.create(sstable);
+            if (indexDescriptor.isIndexEmpty(context))
+                continue;
```

Comment:
This logic is copied from:

https://github.com/datastax/cassandra/blob/a64160f506609b20d8cd34609fe6b7ca255dddf6/src/java/org/apache/cassandra/index/sai/IndexContext.java#L578-L607